controllers/controllers_suite_test.go

```diff
@@ -116,6 +114,8 @@ var _ = BeforeSuite(func(done Done) {
 		},
 	})
 
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
This error checking here seems to be misplaced - `err` is not being assigned anything since the last check. The same thing happens also in `main.go`. Perhaps it's a leftover from a previous state.

Reply:
Removed